routes/main.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const { getAllUrls, getUrl, editUrl, deleteUrl } = require('../controllers/main')
+
+router = express.Router()
+
+
+router.route('/').get(getAllUrls)
+router.route('/:id').get(getUrl).put(editUrl).delete(deleteUrl)
```

Comment:
Changing HTTP methods like this looks cool, but it's generally a good idea. Split them up

Reply:
done on 48ddf7b